documentation/howtos.md

```diff
@@ -59,16 +59,18 @@ ECOMMERCE_REPOS:
     SSH_KEY: '{{ ECOMMERCE_GIT_IDENTITY }}'
 
 # Append to any existing EDXAPP_LMS_ENV_EXTRA:
+# Without this, LMS will error out as the default value
+# doesn't have this currency.
```

Comment:
@lgp171188 the default value of what?

Reply:
@swalladge, the default value of `COURSE_MODE_DEFAULT` defined [here](https://github.com/open-craft/edx-platform/blob/open-release/juniper.3/lms/envs/common.py#L854-L864). This [line](https://github.com/open-craft/edx-platform/blob/open-release/juniper.3/common/djangoapps/course_modes/models.py#L770) errors out when the value passed to the `min()` function is an empty list, which happens when the default course mode doesn't have the default currency.

Reply:
@swalladge, I have made this clear in 2761d4f1dc3f254bb837b4350dc443df458820ce.

Reply:
Ah that's perfect, thanks :+1: